Podfile

```diff
@@ -5,8 +5,8 @@ target 'TrustKeystore' do
 
   pod 'BigInt', inhibit_warnings: true
   pod 'CryptoSwift', '~> 0.10.0'
-  pod 'TrezorCrypto', inhibit_warnings: true
-  pod 'TrustCore', :git=>'https://github.com/TrustWallet/trust-core', :branch=> 'master', inhibit_warnings: true
+  pod 'TrezorCrypto', '~> 0.0.9', inhibit_warnings: true
```

Comment:
Point to master, it’s easier that way to update

Reply:
I will do it later, trust-core branch may have updates